src/diffusers/models/embeddings.py

```diff
@@ -35,10 +35,15 @@ def get_timestep_embedding(
     """
     This matches the implementation in Denoising Diffusion Probabilistic Models: Create sinusoidal timestep embeddings.
 
-    :param timesteps: a 1-D Tensor of N indices, one per batch element.
-                      These may be fractional.
-    :param embedding_dim: the dimension of the output. :param max_period: controls the minimum frequency of the
-    embeddings. :return: an [N x dim] Tensor of positional embeddings.
+    Args
```

Comment:
thanks!
can we also add the other 2 arguments that are missing from the doc string?

Reply:
I can try -- TBH I haven't examined the function closely to really understand what `downscale_freq_shift` and `scale` actually do.